WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -132,15 +132,14 @@ public enum WooAnalyticsStat: String {
     case orderNoteAddButtonTapped               = "add_order_note_add_button_tapped"
     case orderNoteEmailCustomerToggled          = "add_order_note_email_note_to_customer_toggled"
     case orderDetailAddTrackingButtonTapped = "order_detail_tracking_add_tracking_button_tapped"
-    case orderDetailShowBillingTapped           = "order_detail_customer_info_show_billing_tapped"
-    case orderDetailHideBillingTapped           = "order_detail_customer_info_hide_billing_tapped"
     case orderDetailFulfillButtonTapped         = "order_detail_fulfill_order_button_tapped"
-    case orderDetailCustomerEmailTapped         = "order_detail_customer_info_email_menu_email_tapped"
-    case orderDetailCustomerPhoneMenuTapped     = "order_detail_customer_info_phone_menu_tapped"
-    case orderDetailCustomerPhoneOptionTapped   = "order_detail_customer_info_phone_menu_phone_tapped"
-    case orderDetailCustomerSMSOptionTapped     = "order_detail_customer_info_phone_menu_sms_tapped"
+    case orderBillingInfoCustomerEmailTapped         = "order_billing_info_email_menu_email_tapped"
```

Comment:
I am a bit confused here... I don't recall seeing anything about renaming tracks events. I am not saying this is wrong, just that I don't recall it being mentioned.

Reply:
Since all the actions now are located in a new screen, I have thought that using the same events which mentions the old screen was a mistake, then i renamed it using a reference to the new screen (in this way it is easy to understand from which screen an action was triggered). I realize however that behind it there could be some dynamics for which it could be necessary to keep the same name of the events, for example traceability over time. Is this the case?

Reply:
Yes. And sorry, there was no way you would know...

All Tracks events are declared in an internal spreadsheet that guarantees, as you mentioned, persistence over time, as we want to be able to look back at any point in time and check if any action we have taken in the app has any impact in how users interact with it